english/eat-rainbows/right/hand-wash.md

```diff
@@ -0,0 +1 @@
+OMG. Your hands started to hurt, but most of the stains were still on your cloth.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.